src/isolate.rs

```diff
@@ -259,11 +295,18 @@ extern "C" fn pre_dispatch(
 
   if is_sync {
     // Execute op synchronously.
+    let state = isolate.state.clone();
```

Comment:
This is a style preference, but I prefer `Arc::clone(&isolate.state)` over `isolate.state.clone()`. The two are exactly equivalent, however the first makes it clear that it's an Arc that's cloned and not the data itself.

Reply:
Agreed